client/src/pages/private/ParticipantView.js

```diff
@@ -409,14 +422,44 @@ export default () => {
     return 'Change Participant Status';
   };
 
-  const prettifyStatus = (status) => {
-    if (status === 'offer_made') return 'Offer Made';
-    if (status === 'open') return 'Open';
-    if (status === 'prospecting') return 'Prospecting';
-    if (status === 'interviewing') return 'Interviewing';
-    if (status === 'rejected') return 'Rejected';
-    if (status === 'hired') return 'Hired';
-    return status;
+  const prettifyStatus = (status, id) => {
+    let firstStatus = status[0];
+    if (status[0] === 'offer_made') firstStatus = 'Offer Made';
+    if (status[0] === 'open') firstStatus = 'Open';
+    if (status[0] === 'prospecting') firstStatus = 'Prospecting';
+    if (status[0] === 'interviewing') firstStatus = 'Interviewing';
+    if (status[0] === 'rejected') firstStatus = 'Rejected';
+    if (status[0] === 'hired') firstStatus = 'Hired';
+    return <div style={{
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'center',
+    }}>{firstStatus} {status[1] && <ComponentTooltip arrow
```

Comment:
So this will display the same tooltip even if `status` is `['offer_made', 'open']`. Is there a guarantee that this won't happen?

Reply:
this is the line responsible for adding the second status:
`row.status = [previousStatus?.data.previous || item.statusInfos[0].status, 'already_hired'];` (line 195)
For now is guaranteed that the second element is always 'already_hired'.